README.md

```diff
@@ -37,9 +47,32 @@ How to use
     >>> import json
     >>> import urllib2
     >>> from xylose.scielodocument import Journal
-    >>> journal_json = article_json = json.loads(urllib2.urlopen('http://200.136.72.162:7000/api/v1/journal?collection=scl&issn=0103-0663').read())
+    >>> journal_json = json.loads(urllib2.urlopen('http://articlemeta.scielo.org/api/v1/journal?collection=scl&issn=0103-0663').read())
     >>> journal = Journal(journal_json[0])
     >>> journal.title
     u'Revista de Odontologia da Universidade de S\xe3o Paulo'
     >>> journal.scielo_issn
-    u'0103-0663'
\ No newline at end of file
+    u'0103-0663'
+
+**Reading a Issue**
+
+    >>> import json
+    >>> import urllib2
+    >>> from xylose.scielodocument import Issue
+    >>> issue_json = json.loads(urllib2.urlopen('http://articlemeta.scielo.org/api/v1/issue/?collection=scl&code=0103-066319970002').read())
+    >>> issue = Issue(issue_json)
+    >>> issue.journal
+    u'Rev. Odontol. Univ. S\u00e3o Paulo'
+    >>> issue.volume
+    u'11'
+
+
+## Report issues, or request changes
+
+To report problems, bugs, or simply request some new functionality, you can [create a ticket](<https://github.com/scieloorg/xlose/issues>) with your requests.
```

Comment:
Acho que "issues" ficaria melhor do que "problems" aqui. E "bugs" em seguida fica redundante, não? Talvez colocar "To report bugs or request" estaria bom.